styles/stylesheet.css

```diff
@@ -0,0 +1,312 @@
+* {
+  box-sizing: border-box;
+}
+:root {
+  --green: #95be66;
+  --blue: #224971;
+  --grey: #e9e9e9;
+  --lightgrey: #e7e7e7;
+  --headerfont: 'Gelasio';
+  --textfont: 'Open Sans';
+}
+
+/* header */
+a {
+  text-decoration: none;
+  font-family: var(--textfont);
+}
+h1, h2, h3 {
+  color: var(--blue);
+  font-family: var(--headerfont);
+}
+header {
+  width: 100%;
+  background-image: url("../images/header-image.jpg");
+  background-size:cover;
+}
+.header-content-container {
```

Comment:
One improvment I'd like to see is the text below the image flows weird when you make the browser smaller. Maybe make the width of the first colum wider and the middle one smaller to account for smaller viewpoints...